Gemfile.lock

```diff
@@ -41,6 +41,7 @@ GEM
     addressable (2.5.1)
       public_suffix (~> 2.0, >= 2.0.2)
     arel (8.0.0)
+    bcrypt (3.1.11)
```

Comment:
That's fine - all secondary dependencies will appear in this file, even if not explicitly named in Gemfile.